src/e2e/pages/landingPage.js

```diff
@@ -28,12 +28,14 @@ export class LandingPage {
   async goToSignUp () {
     await Promise.all([
       this.page.waitForNavigation(),
-      this.signUpButton.click()
+      this.signUpButton.click(),
+      this.page.waitForURL('**/oauth/**')
     ])
   }
 
   async goToSignIn () {
     this.signInButton.click()
+    await this.page.waitForURL('**/oauth/**')
```

Comment:
Without this line, Playwright would immediate start finding an `input[name="email"]`, and now it would actually immediately find it, since such an input was added to the landing page. However, it _should_ match the one on the Firefox Accounts page :)

(Sorry for the force pushes, took a couple of tries to get the Playwright test right.)